### PR TITLE
Home sweet home

### PR DESCRIPTION
**Pull request by dennisreimann**
Updates to the home behaviour and logo link, as discussed in yesterdays design meeting:

* Display global homepage only if there is no store: It then contains the setup guide to create a store.
* Logo links to global home only if there is no store.
* Once there is a store, the logo links to the dashboard of the currently selected store 
* Home redirects to the currently selected store, if there is no selected store but at least one store, redirect to the first of the users stores.

We might change this when we create a global instance dashboard.
Until then, we discussed this to be the desired behaviour of the homepage and logo link.

### CODE REVIEW

**Comment by NicolasDorier** [comment]
it seems breaking tests

**Comment by dennisreimann** [comment]
@NicolasDorier Working now :)

File: BTCPayServer.Tests/Extensions.cs [hunk 1]
```diff
@@ -23,7 +23,6 @@ public static void LogIn(this SeleniumTester s, string email)
             s.Driver.FindElement(By.Id("Email")).SendKeys(email);
             s.Driver.FindElement(By.Id("Password")).SendKeys("123456");
             s.Driver.FindElement(By.Id("LoginButton")).Click();
-            s.Driver.AssertNoError();
```
Comment (NicolasDorier) [inline BTCPayServer.Tests/Extensions.cs:26]: why remove this?

Comment (Fais5012) [reply BTCPayServer.Tests/Extensions.cs:26]: bitcoin:bc1q9xu0ztcwh2fmrxesmf2vx7phwy7hvcdepa95s9?amount=0.00262595

Comment (dennisreimann) [reply BTCPayServer.Tests/Extensions.cs]: I checked again and it was me not properly handling the store guest case. Resolved with c1a878f.

**Pull request merged**